README.md

```diff
@@ -148,506 +96,126 @@ export const Label: React.FC<PropsWithChildren<LabelProps>> = ({
-
-1) add responsiveVariant definition
-
-```ts
-export const labelVariants = compose({
-    "slots": { 
-        "root": /** @tw */ "text-blue-500",
-        "otherSlot": /** @tw */ "text-blue-500" // here our new slot
-    },
-    "variants": {
-        "size": {
-            "small": /** @tw */ "text-xs",
-            "large": {
-              root: /** @tw */ "text-7xl",
-              otherSlot: /** @tw */ "text-12xl"
-            }
-        },
-        "fontWeight": {
-            xxl: {
-                "root": /** @tw */ "font-extrabold"
-            }
-        }
-    },
-    "defaultVariants": {
-        "size": "small",
-        "fontWeight": "xxl"
     },
-    "responsiveVariants": ["size"]
-})<LabelBaseProps>()
-```
-
-this is important you add responsiveVariants so we will be able to generate tailwind classes.
-
-2) Update tailwind config to have safeList
-
-Safelist in tailwind is a way to force tailwind to add the classes we need to be able to handle responsivness. As tailwind is doing static analysis to add the classes to the output or not. In our case we will not have those available in the code so we need to add them via safeList options:
-
-```ts
-
-export const screens: Screens[] = ["sm", "md", "lg", "xl"]
-
-export default {
-  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx,mdx}"],
-  theme: {
-    extend: {
-      screens
-    }
   },
-  safelist: generateSafeList([labelVariants], screens) // import your variant definition the generateSafeList is taking care of the rest
-};
-```
-
-Note: that we have added a screens property here. Make sure to add it here also. We have an on going issue to make the screens overridable in typescript and also in the config. Adding inside the config is easy but we do not have typescript auto complete working yet. So for now please use the default tailwind screen definition
-
-3) Simple Usage
-
-Now everything is defined and you understand how it works lets see the usage:
-
-```ts
-root({
-  "size": {
-      "initial": "small", // this is mandatory to have an intiial option to respect mobile first approach
-      "md": "large"
+  defaultVariants: {
+    size: "small",
   },
-  "fontWeight": "xxl" // this would not accept a responsive value as you didnt' define it in the responsiveVariants array
-})
-```
+});
 
-With our latest definition
-
-```ts
-"slots": { 
-  "root": /** @tw */ "text-blue-500",
-  "otherSlot": /** @tw */ "text-blue-500" // here our new slot
-},
-"variants": {
-    "size": {
-        "small": /** @tw */ "text-xs",
-        "large": {
-          root: /** @tw */ "text-7xl",
-          otherSlot: /** @tw */ "text-12xl"
-        }
-    },
-    "fontWeight": {
-        xxl: {
-            "root": /** @tw */ "font-extrabold"
-        }
-    }
-},
-"defaultVariants": {
-  "size": "small",
-  "fontWeight": "xxl"
-},
-"responsiveVariants": ["size"]
+// Usage
+const { root, header, body } = cardVariants({ size: "large" });
```

Comment:
I'll add some JSX to show how you use `root`, `header` and `body`

Reply:
thanks for the comment the example was actually wrong